types.go

```diff
@@ -19,8 +19,8 @@ type Prefix []byte
 // hash from the one used for interior nodes.
 type KeyValuePair struct {
 	_struct bool        `codec:",toarray"`
-	Key     Hash        `codec:"key"`
-	Value   interface{} `codec:"value"`
+	Key     Hash        `codec:"k"`
+	Value   interface{} `codec:"v"`
```

Comment:
To fill in our discussion from Slack, the issue with the `Value []bytes` proposal is that it costs 2 bytes more per Value, as long as the value is less than 256 bytes.  Above that, it costs 3 bytes more per Value.  I guess this isn't a big deal in the scheme of things.